e2etests/pkg/config/from_containers.go

```diff
@@ -25,33 +32,57 @@ type PeersConfig struct {
 	Secrets []corev1.Secret
 }
 
+func (pc PeersConfig) Peers() []Peer {
+	return append(pc.PeersV4, pc.PeersV6...)
+}
+
+func (pc PeersConfig) FRRK8sNeibghbor() []frrk8sv1beta1.Neighbor {
```

Comment:
typo: Neibghor

Reply:
also, it should be Neighbors, and also I think this can be a function and not a method. Something like `NeighborsFrom`.